engine.c

```diff
@@ -19,6 +19,7 @@ const char engine_ini_defaults[] = {
 	"expose_php = 0\n"
 	"default_mimetype =\n"
 	"html_errors = 0\n"
+	"error_reporting = E_ALL\n"
```

Comment:
Hrm, this didn't have the desired effect of fixing the tests; unsure what is causing them to fail on Travis CI, then.